.env.example

```diff
@@ -50,4 +50,5 @@ ISSUER_ID=Issuer ID
 # Possible options: GRID_OPERATOR and LOCATION, should be comma separated
 DEVICE_PROPERTIES_ENABLED=GRID_OPERATOR,LOCATION
 # Amount of energy in single certificate 
-ENERGY_PER_UNIT=1000000
\ No newline at end of file
+ENERGY_PER_UNIT=1000000
+DEFAULT_ENERGY_IN_BASE_UNIT=1;
```

Comment:
Why not use `ENERGY_PER_UNIT=1000000`? 1000000Wh is 1MWh which 1 from this newly added variable. Note that all operations in the system are denominated in Wh.

@JosephBagaric what's your opinion?

Reply:
You mean `DEFAULT_ENERGY_IN_BASE_UNIT`?

I assume "base unit" is Wh, and "display unit" is MWh.

Depends on how this is being used. I see that it's set as a default for an energy input field.
I would set it to 1 000 000 (1 MWh), as we only use and show MWhs in the UI for now.